openedx.yaml

```diff
@@ -7,9 +7,3 @@ tags:
     - tools
 oeps:
     oep-2: True
```

Comment:
We should add this line to get the repo tagged with releases.
```
openedx-release: {ref: release}
```

Reply:
Will do.